webapp/channels/src/components/view_user_group_modal/view_user_group_list_item/view_user_group_list_item.tsx

```diff
@@ -70,9 +70,9 @@ const ViewUserGroupListItem = (props: Props) => {
                     aria-label='Close'
                     onClick={removeUserFromGroup}
                 >
-                    <LocalizedIcon
+                    <i
                         className='icon icon-trash-can-outline'
-                        ariaLabel={{id: t('user_groups_modal.goBackLabel'), defaultMessage: 'Back'}}
+                        aria-label={formatMessage({id: 'user_groups_modal.goBackLabel', defaultMessage: 'Back'})}
```

Comment:
Hey @sjcode99 can you remove aria label from this icon because the parent button has Aria label already

Reply:
@M-ZubairAhmed , should I remove the only aria-label attribute from <i/> tag?

Reply:
Yes my reasoning is that when a button is focused the screen reader will read its text or Aria label and the icon underneath Aria label is not required there.